SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs

```diff
@@ -270,7 +271,8 @@ public static LastUpdated ToDomain(this dbLastUpdated update)
                 LastEdited = caseSubmission.EditHistory.Last().EditTime,
                 CompletedSteps = caseSubmission.FormAnswers.Count,
                 Deleted = caseSubmission.Deleted ?? false,
-                DeletionDetails = includeDeletionDetails ? caseSubmission.DeletionDetails : null
+                DeletionDetails = includeDeletionDetails ? caseSubmission.DeletionDetails : null,
+                PinnedAt = caseSubmission.PinnedAt
```

Comment:
is pinned timedate returned in specific GETs of a particular case, as well as the list GET?